Description: Disable test failing in CI on Linux
Last-Update: 2022-08-02

--- test/regress_dns.c.orig	2022-08-02 16:31:35.821067863 -0700
+++ test/regress_dns.c	2022-08-02 16:32:46.128182960 -0700
@@ -2479,8 +2479,6 @@
 
 	{ "getaddrinfo_async", test_getaddrinfo_async,
 	  TT_FORK|TT_NEED_BASE, &basic_setup, (char*)"" },
-	{ "getaddrinfo_cancel_stress", test_getaddrinfo_async_cancel_stress,
-	  TT_FORK, NULL, NULL },
 
 #ifdef EVENT_SET_MEM_FUNCTIONS_IMPLEMENTED
 	{ "leak_shutdown", test_dbg_leak_shutdown, TT_FORK, &testleak_funcs, NULL },
